app/fix-component-types.sh

```
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# React.FC削除後のコンポーネントに型引数と戻り値型を追加
+
+cd /Users/masakazuiwahara/Development/tempo-ai/app/src/components
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)